hack/scripts/generate-cli-unit-tests-report.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# Copyright 2023 VMware, Inc. All Rights Reserved.
```

Comment:
Did you write this script yourself or did you find it on the internet?
If it is taken from the internet, it is most probably fine, but we cannot put a VMware copyright

Reply:
i wrote the script